0000-product.md

```diff
@@ -0,0 +1,468 @@
+- Feature Name: Product
+- Start Date: 5/13/19
+- RFC PR: [#5](https://github.com/hyperledger/grid-rfcs/pull/5)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes a GS1-compatible implementation of Product for Hyperledger
+Grid.  GS1 is a widely used data standard in enterprises and, given that
```

Comment:
This RFC proposes a generic and extensible framework for a Hyperledger Grid Product entity as well as a more specific - and fully encapsulated - GS1 compliant Product.

_(The point being this isn't just about GS1 Product, it's about Grid Product, but the only more specific Product being proposed here is the GS1 flavor.)_

Reply:
✅